src/initialization/FGInitialCondition.cpp

```diff
@@ -1519,7 +1524,8 @@ void FGInitialCondition::bind(FGPropertyManager* PropertyManager)
   PropertyManager->Tie("ic/vw-down-fps", this,
                        &FGInitialCondition::GetWindDFpsIC);
   PropertyManager->Tie("ic/vw-mag-fps", this,
-                       &FGInitialCondition::GetWindFpsIC);
+                       &FGInitialCondition::GetWindFpsIC,
+                       &FGInitialCondition::SetWindMagFpsIC);
```

Comment:
The getter and setter have dissimilar names: should be `SetWindFpsIC()` ?

Reply:
Yep, did wonder about that a bit. Matched the new setter to match the name of the existing setter that is identical except for having `Kts` in the name.

Looking at it again I'm actually inclined to rename the getter from `GetWindFpsIC()` to `GetWindMagFpsIC()` to make it clear it's returning the magnitude as opposed to a vector or some vector component.

Reply:
> Looking at it again I'm actually inclined to rename the getter from `GetWindFpsIC()` to `GetWindMagFpsIC()` to make it clear it's returning the magnitude as opposed to a vector or some vector component.

Looks good to me.